thali/thalireplicationmanager.js

```diff
@@ -104,7 +106,10 @@ ThaliReplicationManager.prototype.stop = function () {
   }.bind(this));
 };
 
-function networkChanged(status) {
+/**
+ * Reacts to the networkChanged event to start or stop based upon its current status.
+ */
+ThaliReplicationManager.prototype._networkChanged = function (status) {
```

Comment:
Purely as an example of defensive programming would it make sense to put a return after line 114?